test/tests/functional/pbs_qstat.py

```diff
@@ -47,31 +47,34 @@ class QstatTest(TestFunctional):
     """
 
     def setUp(self):
-        if len(self.servers) != 2 or self.server.client in self.servers:
-            self.skipTest("This test needs two servers and one client")
+        if self._testMethodName == "test_qstat_req_server":
```

Comment:
The default PTL setUp will setup multiple servers if pbs_benchpress is provided with server names, but I needed to change the config to also add 'nomom' information before calling setUp, otherwise it fails. So, I had to do this in setUp itself as opposed to inside the test.

Reply:
Should this just be moved to a TestQstat2Servers test suite? 
That way, there doesn't have to check the name of the test.

Reply:
Thanks for the suggestion Vincent. Since this testsuite has only this one test, we can just rename this very testsuite and then I won't need the testcase name check. What do you think @bhroam ?

Reply:
Ok, I went ahead and renamed the testsuite since it had just this one test, let me know if it looks okay to you guys.